packages/libxshmfence.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class Libxshmfence < Package
+  description 'libxshmfence: A library that exposes a event API on top of Linux futexes'
```

Comment:
Can you please change this to `description 'A library that exposes a event API on top of Linux futexes'`?  The reason is `crew search libxshmfence` will return `libxshmfence: libxshmfence: A library that exposes a event API on top of Linux futexes`.  A bit redundant. :)